packages/e2e-tests/specs/experiments/site-editor-export.test.js

```diff
@@ -0,0 +1,64 @@
+/**
+ * External dependencies
+ */
+import fs from 'fs';
+import path from 'path';
+import os from 'os';
+
+/**
+ * WordPress dependencies
+ */
+import { trashAllPosts, activateTheme } from '@wordpress/e2e-test-utils';
+
+/**
+ * Internal dependencies
+ */
+import { siteEditor } from '../../experimental-features';
+
+async function waitForFileExists( filePath, timeout = 10000 ) {
+	const start = Date.now();
+	while ( ! fs.existsSync( filePath ) ) {
+		await delay( 1000 );
+		if ( Date.now() - start > timeout ) {
+			throw Error( 'waitFile timeout' );
+		}
+	}
+}
+
+function delay( time ) {
```

Comment:
This is just a duplication of page.waitFor it seems.

Reply:
`page.waitFor` is deprecated, so I'd rather avoid introducing it in new tests. I'm thinking about replacing `delay` with `page.waitForTimeout`.

Reply:
We may have to update our waitFor lint rule to also target the usage of waitForTimeout